src/hls_lpdaac_reconciliation/response/index.py

```diff
@@ -163,15 +160,14 @@ def process_report(
     """
 
     granule_ids_per_status_per_collection = {}
+
     for collection_id, (n_files, granule_ids) in group_granule_ids(report).items():
         print(
-            f"{len(granule_ids)} granule ({n_files} file) missing from {collection_id}"
+            f"{len(granule_ids)} granule(s) ({n_files} file(s)) "
+            f"missing from {collection_id}"
         )
 
-        short_name, version = decode_collection_id(collection_id)
```

Comment:
nit - is this function now dead code / something to delete?

Reply:
Good find! I missed that, obviously. Of course, if we were using ruff, it likely would have alerted me about this, so it looks like we should do a bit of modernization in this repo, but for a separate PR, although, I'll remove the dead code.